configs/MATEKF405WINGV2/config.h

```diff
@@ -0,0 +1,125 @@
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        MATEKF405WINGV2
+#define MANUFACTURER_ID   MTKS
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_SDCARD
+
+#ifndef USE_WING
+#define USE_WING
+#endif
+
+#ifndef USE_SERVOS
+#define USE_SERVOS
+#endif
+
+#define BEEPER_PIN           PC15
+#define MOTOR1_PIN           PB7
+#define MOTOR2_PIN           PB6
+#define SERVO1_PIN           PB0
+#define SERVO2_PIN           PB1
+#define SERVO3_PIN           PC8
+#define SERVO4_PIN           PC9
+#define SERVO5_PIN           PB14
+#define SERVO6_PIN           PB15
+#define RX_PPM_PIN           PA3
+#define LED_STRIP_PIN        PA15
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PC10
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PC11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C2_SCL_PIN         PB10
+#define I2C1_SDA_PIN         PB9
+#define I2C2_SDA_PIN         PB11
+#define LED0_PIN             PA14
+#define LED1_PIN             PA13
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PB3
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PC2
+#define SPI3_SDI_PIN         PB4
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PC3
+#define SPI3_SDO_PIN         PB5
+#define ADC_VBAT_PIN         PC0
+#define ADC_RSSI_PIN         PC5
+#define ADC_CURR_PIN         PC1
+#define SDCARD_SPI_CS_PIN    PC14
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+#define USB_DETECT_PIN       PC13
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PB7 , 1,  0) \
+    TIMER_PIN_MAP( 1, PB6 , 1,  0) \
+    TIMER_PIN_MAP( 2, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 3, PB1 , 2,  0) \
+    TIMER_PIN_MAP( 4, PC8 , 2,  1) \
+    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 6, PB14, 3, -1) \
+    TIMER_PIN_MAP( 7, PB15, 3, -1) \
```

Comment:
Servo does not need DMA
```suggestion
    TIMER_PIN_MAP( 2, PB0 , 2,  -1) \
    TIMER_PIN_MAP( 3, PB1 , 2,  -1) \
    TIMER_PIN_MAP( 4, PC8 , 2,  -1) \
    TIMER_PIN_MAP( 5, PC9 , 2,  -1) \
    TIMER_PIN_MAP( 6, PB14, 3, -1) \
    TIMER_PIN_MAP( 7, PB15, 3, -1) \
```